localstack-core/localstack/services/apigateway/next_gen/execute_api/handlers/integration_response.py

```diff
@@ -203,7 +203,7 @@ def get_response_template(
             return ""
 
         # The invocation request header is used to find the right response templated
-        accepts = request["raw_headers"].getlist("accept")
+        accepts = request["headers"].getlist("accept")
```

Comment:
it seems for the `render_request_template_mapping` (btw, it seems the name was copy pasted from the integration request) we didn't call `dict(context.invocation_request.get("headers", {})` for the `MappingTemplateParams` on line 233

Reply:
It seems like we might not need to, since the VTL template is calling .get(), it appears we are returning the proper values.

We can probably update the render_request_template_mapping as well, and safe a bit on efficiency by not converting the headers to dict for the templates.